src/lib/extension.ts

```diff
@@ -1 +1,3 @@
 export const IDENTIFIER = 'aha-develop.azure-devops-repos';
+export const LEARN_MORE_URL = 'https://github.com/aha-develop/azure-devops-repos';
+export const ICON = aha.iconForExtensionIdentifier(IDENTIFIER);
```

Comment:
This line causes an exception in the endpoint handler.
```
TypeError: aha.iconForExtensionIdentifier is not a function
JavaScript at src/lib/extension.ts:3:25
JavaScript at aha-develop.azure-devops-repos.webhook:247:3
JavaScript at aha-develop.azure-devops-repos.webhook:248:3
JavaScript at Aha.defineExtension (ContributionRegistry.js:50:7)
JavaScript at aha-develop.azure-devops-repos.webhook:1:34
JavaScript at aha-develop.azure-devops-repos.webhook:249:4
```

Reply:
@trydionel this should be fixed now

Reply:
Fixed